### ISSUE TITLE
Picks demo page failing Accessibility Insights FastPass

### ISSUE THREAD

**Issue by chuanqisun** [microsoft/fluentui#21266]
### Environment Information

- **Package version(s)**: 8.49.4
- **Browser and OS versions**: Edge  97.0.1072.55 Windows 11

### Describe the issue:

Performing FastPass scan on the [Pickers demo page](https://developer.microsoft.com/en-us/fluentui#/controls/web/pickers) result in 5 failures. See attached scan results

[Accessibility Insights for Web automated checks result.zip](https://github.com/microsoft/fluentui/files/7856918/Accessibility.Insights.for.Web.automated.checks.result.zip)

### Please provide a reproduction of the issue in a codepen:

The issue is on the demo page.

#### Actual behavior:

![image](https://user-images.githubusercontent.com/1895289/149201837-dd0d226b-6910-4b4f-8ceb-df1986de6101.png)

![image](https://user-images.githubusercontent.com/1895289/149201880-c0b03933-01e4-44e1-89fb-6a94a98c7438.png)

#### Expected behavior:

No failures should be found by FastPass

**Comment by gouttierre** [comment]
Thanks for filing this issue with us. To set expectations, the developers will review this issue once capacity allows.

@spmonahan - Would you be able to confirm if this is a regression, or if this behavior is an issue? 
cc @smhigley

**Comment by smhigley** [comment]
Closing, this was fixed in an earlier PR.
